integration/presto/src/main/java/org/apache/carbondata/presto/readers/AbstractStreamReader.java

```diff
@@ -1,66 +0,0 @@
-/*
```

Comment:
no need this class now, as we have updated the interface and now streamReaders extends  CarbonColumnVectorImpl, all functionalities are captured within the reader